libcontainer/intelrdt/intelrdt.go

```diff
@@ -738,6 +745,12 @@ func (m *intelRdtManager) Set(container *configs.Config) error {
 		l3CacheSchema := container.IntelRdt.L3CacheSchema
 		memBwSchema := container.IntelRdt.MemBwSchema
 
+		// TODO: verify that l3CacheSchema and/or memBwSchema match with
+		// existing schemata if ClosID has been specified. This is a more
+		// involved than reading the file and doing plain string comparison as
+		// the value written in does not necessarily match what gets read out
+		// (leading zeros, cache id ordering etc).
```

Comment:
Maybe it makes sense to relax the runtime spec and say "runtime MAY check" rather than "must check", because it if is a must and you're not doing it, you're not conformant.

Reply:
Might be, but I'm not entirely sure as it leads to inconsistency. User asking for `ClosID=foo,  L3Schema=bar` might get what she asks for or then something totally different.

Anyways, I think that should be debated in runtime-spec repo instead of runc.